app/Livewire/Project/Database/BackupEdit.php

```
@@ -216,6 +236,15 @@ private function customValidate()
         if (! $isValid) {
             throw new \Exception('Invalid Cron / Human expression');
         }
+
+        // Validate pgBackRest-specific requirements
+        if ($this->backup->backup_method === 'pgbackrest') {
+            // Only PostgreSQL databases support pgBackRest
+            if ($this->backup->database->type() !== 'standalone-postgresql') {
+                throw new \Exception('pgBackRest is only supported for PostgreSQL databases');
```

PHPMD (2.15.0)

Missing class import via use statement (line '244', column '27'). (undefined)

(MissingImport)